micro-ui/web/micro-ui-internals/packages/modules/utilities/src/pages/employee/index.js

```
@@ -51,12 +54,21 @@ const App = ({ path, stateCode, userType, tenants }) => {
         <Route path={`${path}/playground/form-composer`}>
         <FormExplorer></FormExplorer>
         </Route>
+        <Route path={`${path}/playground/form-composer`}>
+        <FormExplorerCitizen></FormExplorerCitizen>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)